tsconfig.json

```diff
@@ -26,7 +26,7 @@
     "index.js",
     "bin",
     "lib",
-    "tests"
+    "test"
```

Comment:
In my previous PR, I had a typo in the directory name. As a result, `tsc` did not check any test files.